PZ/WebClient/views.py

```diff
@@ -101,8 +101,12 @@ def hosts(request, monitor_id):
     current_monitor = Monitors.objects.get(id=monitor_id)
     c = Connector(urljoin(current_monitor.monitor_domain, 'resources'))
 
+    if request.method == 'GET':
+        search_query = request.GET.get('name', None)
+        c._payload = {"name": search_query}

```

Comment:
`Connector` has payload as property, so you should use `.payload` instead of `._payload`